tests/setup_test_environment.sh.in

```diff
@@ -32,20 +39,22 @@ case "$PROG_DIR" in
         ;;
 esac
```

Comment:
The return values of `echo` and `sed` are being lost here, but it's probably not a big deal.